_scss/_utilities.scss

```diff
@@ -58,6 +58,9 @@ i.fa.fa-outdent {
     padding: 0px 15px 10px 15px;
 }
 
+.image-inline img {
```

Comment:
try changing this to

```css
img.image-inline {
    display: inline;
}
```

The style your currently defined would match an `<img>` element that's a child element of an element that has the `.image-inline` class, for example;

```html
<p class="image-inline"><img src="foobar.png /></p>
```